doc/book/admin/access_control.rst

```diff
@@ -1,377 +1,1069 @@
 .. _authentication:
+.. _access_control:
 
-================================================================================
 Access control
-================================================================================
+==============
```

Comment:
Idea: use the words `role-based access control system/model` explicitly somewhere in the intro paragraph. This is a familiar term, it helps readers understand general concepts easier.

Reply:
Privileges can be assigned directly to a user. So, `role-based access control` doesn't cover all possible capabilities of Tarantool access control system.